packages/playout-gateway/src/connector.ts

```diff
@@ -81,4 +93,46 @@ export class Connector {
 			process.exit(0)
 		}, 10 * 1000)
 	}
+
+	/* eslint-disable @typescript-eslint/no-inferrable-types */
+	/* eslint-disable @typescript-eslint/no-empty-function */
+	private setupKoaEndpoints(): void {
+		const koaRouter: KoaRouter = new KoaRouter()
+		this.setupDevicesMakeReadyEndpoint(koaRouter)
+		this.setupDevicesStandDownEndpoint(koaRouter)
+
+		const port: number = 3009
+		const koaApp: Koa = new Koa()
+
+		koaApp.use(koaRouter.routes()).use(koaRouter.allowedMethods())
+		koaApp.listen(port, () => {})
+	}
+
+	/* eslint-disable @typescript-eslint/prefer-as-const */
+	private setupDevicesMakeReadyEndpoint(router: KoaRouter): KoaRouter {
+		const okToDestroyStuffQueryParameterName: 'okToDestroyStuff' = 'okToDestroyStuff'
+		const activeRundownIdParameterName: 'activeRundownId' = 'activeRundownId'
+
+		router.post(`/devicesMakeReady`, async (context: KoaContext): Promise<void> => {
+			const okToDestroyStuff: boolean = context.query[okToDestroyStuffQueryParameterName] === 'true'
+			const activeRundownId: string | string[] | undefined = context.query[activeRundownIdParameterName]
```

Comment:
Why can it be a `string[]`?

Reply:
I don't know. It seems that that is the type from Koa.
If I don't include `string[]` then TypeScript complains about non-matching types.